# Keynote I
## Katie Pollard
## Population genetic models for human associated microbes

* precision medicine 2.0 - microbiome - our other genome
* human associated microbes - trillions of genomes, 5lbs of body weight, species abundances vary
* immense within species diversity
* strains of same species have different genes
* genetic variation within a person over time, genetic variation within a single population inside body
* genes from same species have different mutations (many non-synonymous)
* 5-50% or CNVs
* SNP / 50-500bp
* 0-10 distinct lineages per host, one or two dominant lineages
* private SNPs unique to host
* accurate taxonomy not enough - form and function change through genetic modifications, presence of different enzymes important not necessarily the specific species
* measuring microbiome genetic variation can tell us about microbiome evolution, strain gene tracking, human evolution, genomic technologies, clinical decision making/precision therapies
* Most species have no genome
* goal- catalog of common and rare microbial SNPs- CRISPR based enrichment
* Phasing shotgun metagenomes - can use cutoffs to assign major/minor allele, sc-seq, eventually reference pannels
* mechanisms changing alleles
    - ecological: species fluctuations, invasion
    - evolution: de novo mutation, introgression
* Read cloud barcoded droplet sequencing resolves haplotypes (called read clouds because droplets end up have around 10 different cells of different species)
    - Data 10x genomics sequencing, 1 host, weekly sampling for 1 year
    - confirms observations from phased metagenomes
    - Observe high linkage within sample and over time
    - dynamic genetic responses to antibiotics - dominant strain could be purged but recovered, rare strain could take over, gene sweep on background of dominant strain
* strains replace over lifetime - cohabitating siblings including twins have similar strains, adult twins living apart around 40 years have strains that are no more similar than unrelated hosts
* strain tracking - many infant gut strains not from mother
    - species converge to mom's species over time, but same trend with unrelated mothers
    - ==maturation of the infant microbiome, track strains with rare or private mutation --> strains get less similar to moms over time
* transmission threshold - seeing a snp in mother and child, what is the chance that the mutation is recurrent?
* microbial phylogeny biases trait-microbial variant association tests
